src/sentry/models/groupinbox.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import absolute_import
+
+from enum import Enum
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import FlexibleForeignKey, Model, JSONField
+
+
+class GroupInboxReason(Enum):
+    NEW = 0
+    UNIGNORED = 1
+    REGRESSION = 2
+
+
+class GroupInbox(Model):
+    """
+    A Group that is in the inbox.
+    """
+
+    __core__ = False
+
+    group = FlexibleForeignKey("sentry.Group", unique=True, db_constraint=False)
```

Comment:
Do you need organization or project links on this model so that you can get the inbox state for a project/org?